### ISSUE TITLE
Add auto update in app feature

### ISSUE THREAD

**Issue by HJMBgit84** [rafaelvcaetano/melonDS-android#415]
I see a lot of users reporting crashes with different previos versions of melonDS, maybe that can be avoided if the app when it opens then notify about the most recent latest version to make users a little force to update and report crashes of the most recent version not versions like the 1.3 that was long ago.

It is just a proposal from me that I am just another user.

:)

**Comment by rafaelvcaetano** [comment]
Yeah, I guess it's about time I implement something like that. The next release should bring the app to the Play Store (and probably f-droid) which should mitigate the problem. But I also want to keep releasing updates here on GitHub so this still makes sense.

**Comment by HJMBgit84** [comment]
That sounds really cool 😎, great to know that, thanks for all your great work 😊👍🏻
